1.10.7/buster/Dockerfile

```diff
@@ -21,12 +21,14 @@ LABEL maintainer="Astronomer <humans@astronomer.io>"
 
 ARG ASTRONOMER_USER="astro"
 ARG ASTRONOMER_UID="50000"
+ARG VERSION="1.10.7-7"
 
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.distro="debian"
 LABEL io.astronomer.docker.module="airflow"
 LABEL io.astronomer.docker.component="airflow"
 LABEL io.astronomer.docker.uid="${ASTRONOMER_UID}"
+LABEL io.astronomer.docker.ac.version="${VERSION}"
```

Comment:
This one (and the arg) shouldn't moved up here - it will blow the docker layer cache ntoo early

Reply:
I had updated that in the PR, can you double check the PR please or the last fixup commit